README.md

```diff
@@ -253,9 +254,62 @@ Map<String,String> queryParams = new HashMap<String,String>(){{
 CategoryResponse getCategories = butterClient.getCategory("java",queryParams);
 ```
 
+## Tags
+
+### List Tags
+
+Listing Tags returns a [TagsResponse](#tagsresponse-class) object. This object consists of a List&lt;[Tag](#Tag-class)&gt;
+
+#### getTags() Parameters
+| Parameter|Description|
+| ---|---|
+| queryParams | Map of additional Query Parameters|
+
+#### getTags() Parameters
+
+| Query Parameter|Description|
+| ---|---|
+| include |If value is `recent_posts`,  will return recent posts along with tags|
+
+#### Examples
+
+```java
+Map<String,String> queryParams = new HashMap<String,String>(){{
+    add("include","recent_posts");
+    ...
+}}
+TagsResponse getTags = butterClient.getTags(queryParams);
+```
+
+### Retrieve a Single Tag
+
+Retrieving an author returns an [TagResponse](#tagresponse-class) object. This object consists of single [Tag](#tag-class) if found
+
+#### getTag() Parameters
+
+| Parameter|Description|
+| ---|---|
+|slug|The slug of the tag to be retrieved.|
+| queryParams | Map of additional Query Parameters|
+
+
+| Parameter|Description|
+| ---|---|
+| include |If value is `recent_posts`,  will return recent posts along with tag|
+
+#### Examples
+
+```java
+Map<String,String> queryParams = new HashMap<String,String>(){{
+    add("include","recent_posts");
+    ...
+}}
+TagResponse getTag = butterClient.getTag("java",queryParams);
+```
+
 ## Feeds
 
-Each of the feeds methods returns an [Document](https://docs.oracle.com/javase/7/docs/api/org/w3c/dom/Document.html).
+Each of the feeds methods returns a [Document](https://docs.oracle.com/javase/7/docs/api/org/w3c/dom/Document.html).
```

Comment:
This could be (optionally) a separate commit, `Fix typo`, to keep commits granular, I think.